Changelog.md

```diff
@@ -3,8 +3,9 @@
 ## 0.0.29
 
 - API based components support
-  - Completion and hover for properties
-  - Completion for API name in Camel URI path parameter
+  - Completion and hover for properties
```

Comment:
I would suggest to use the term `hover documentation` instead of just `hover`.

Reply:
this line is marked as modified because of end of lien but not part modified text. Will handle it separately

Reply:
https://github.com/camel-tooling/camel-lsp-client-vscode/pull/470